tests/virt/upgrade/test_upgrade_virt.py

```diff
@@ -89,9 +91,8 @@ def test_vm_ssh_before_upgrade(self, vms_for_upgrade):
     )
     def test_migration_before_upgrade(self, skip_if_no_common_cpu, vms_for_upgrade):
         for vm in vms_for_upgrade:
-            if vm_is_not_migrateable(vm=vm):
-                continue
-            migrate_vm_and_verify(vm=vm, wait_for_interfaces=False, check_ssh_connectivity=False)
+            if vm_is_migrateable(vm=vm):
+                migrate_vm_and_verify(vm=vm, wait_for_interfaces=False, check_ssh_connectivity=False)
```

Comment:
I know it was here before, but shouldn't we wait for ssh after migration, before proceeding with upgrade?

Reply:
+1

Reply:
in real - it does not matter, we anyway check ssh after the upgrade